package-lock.json

```diff
@@ -1,4737 +1,8 @@
 {
   "name": "ide-vscode",
   "version": "2.5.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
?

Reply:
I had to upgrade npm: `npm install -g npm@latest`